lesson-04/remix-site/app/routes/recipes/new.jsx

```diff
@@ -5,19 +5,21 @@ export const action = async ({ request }) => {
   const form = await request.formData();
   const title = form.get("title");
   const body = form.get("body");
+  const ingredients = form.get("ingredients");
+  const seperatedIngredients = ingredients.split(",");
```

Comment:
**Suggestion**
Could use `.map((string) => string.trim());`
To map and trim unwanted spaces and so on 😁